languages/javascript/exercises/concept/array-destructuring/.docs/after.md

```diff
@@ -0,0 +1,82 @@
+Array destructuring assignment is a concise way of extracting values from an array. Its syntax is similar to an array literal expression, but on the left-hand side of the assignment instead of the right.
```

Comment:
Maybe we can link back to MDN on `destructuring assignment`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment, same for `array literal expression`.

Reply:
Done.